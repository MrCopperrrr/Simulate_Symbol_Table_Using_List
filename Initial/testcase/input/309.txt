INSERT m string
PRINT
BEGIN
INSERT n number
PRINT
INSERT o string
BEGIN
INSERT p number
PRINT
END
PRINT
END
PRINT