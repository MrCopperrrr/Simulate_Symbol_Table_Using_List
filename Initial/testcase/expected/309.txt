success
m//0
success
m//0 n//1
success
success
m//0 n//1 o//1 p//2
m//0 n//1 o//1
m//0